lib/models/users.js

```diff
@@ -24,7 +24,7 @@ module.exports = function (sequelize, DataTypes) {
                 unique: 'email'
             },
             mobileNo: {
-                field: '',
+                field: 'mobile_no',
```

Comment:
So this is nice  
but this change should be made in former PR only, since your prev PR was reg modals and migrations, this one is solely for seeding the db 

also think of it as merging your former PR into production, and we are assuming whatever bugs or stuff we missed, upcoming PR will cover it, but until the latter PR gets merged, the production be dealing with that very issue right